rc/tools/git.kak

```diff
@@ -134,7 +131,12 @@ define-command -params 1.. \
                        cmd | getline dates[sha]
                        close(cmd)
                   }
-                  END { send_flags(); }'
+                  END {
+                      update_flags()
+                      cmd = "kak -p " ENVIRON["kak_session"]
+                      print "set-option -add buffer=" ENVIRON["kak_bufname"] " git_blame_flags " flags | cmd
+                      close(cmd)
```

Comment:
This is probably fine for now, but do note that makes us lose the incremental updates (some line flags are already shown before git-blame exits) which can be noticeable on large files in a large repo.

Since we are a long-running process (`> /dev/null 2>&1 < /dev/null &`) our only way to communicate with Kakoune is `kak -p`.
(although we could also write to Kakoune's command socket directly, like [kakoune-smooth-scroll]( https://github.com/caksoylar/kakoune-smooth-scroll/blob/master/smooth-scroll.py#L15)).

I think between running `kak -p` for every new blame and it onlly once, a reasonable middle ground would be to run it once per second.
We just need an atomic buffer to keep our `set-option -add` commands; I'm not sure how to do that with shell/awk. I guess we could create a bunch of temporary files but that might defeat the point of this patch.